src/main/kotlin/tanvd/grazi/language/LangTool.kt

```diff
@@ -20,7 +20,9 @@ object LangTool : GraziLifecycle {
                 lang.configure(this)
                 disableRules(allActiveRules.map { it.id }.filter { it in GraziConfig.state.userDisabledRules })
                 //Spellcheck will be done by Grazi spellchecker
-                disableRules(allActiveRules.filter { it.isDictionaryBasedSpellingRule }.map { it.id })
+                if (arrayOf(Lang.AMERICAN_ENGLISH, Lang.BRITISH_ENGLISH).any { it == lang }) {
```

Comment:
You should check if this is is English at all, not if it is one of a dialects

Reply:
Done